monkey/tests/unit_tests/monkey_island/cc/repository/test_mongo_credentials_repository.py

```diff
@@ -0,0 +1,134 @@
+import mongomock
+import pytest
+
+from common.credentials import Credentials
+from monkey_island.cc.repository import MongoCredentialsRepository
+
+USER1 = "test_user_1"
+USER2 = "test_user_2"
+USER3 = "test_user_3"
+PASSWORD = "12435"
+PASSWORD2 = "password"
+PASSWORD3 = "lozinka"
+LM_HASH = "AEBD4DE384C7EC43AAD3B435B51404EE"
+NT_HASH = "7A21990FCD3D759941E45C490F143D5F"
+PUBLIC_KEY = "MY_PUBLIC_KEY"
+PRIVATE_KEY = "MY_PRIVATE_KEY"
+
+CREDENTIALS_DICT_1 = {
+    "identities": [
+        {"credential_type": "USERNAME", "username": USER1},
+        {"credential_type": "USERNAME", "username": USER2},
+    ],
+    "secrets": [
+        {"credential_type": "PASSWORD", "password": PASSWORD},
+        {"credential_type": "LM_HASH", "lm_hash": LM_HASH},
+        {"credential_type": "NT_HASH", "nt_hash": NT_HASH},
+        {
+            "credential_type": "SSH_KEYPAIR",
+            "public_key": PUBLIC_KEY,
+            "private_key": PRIVATE_KEY,
+        },
+    ],
+}
+
+CREDENTIALS_DICT_2 = {
+    "identities": [
+        {"credential_type": "USERNAME", "username": USER3},
+    ],
+    "secrets": [
+        {"credential_type": "PASSWORD", "password": PASSWORD2},
+        {"credential_type": "PASSWORD", "password": PASSWORD3},
+    ],
+}
+
+CONFIGURED_CREDENTIALS = [Credentials.from_mapping(CREDENTIALS_DICT_1)]
+
+STOLEN_CREDENTIALS = [Credentials.from_mapping(CREDENTIALS_DICT_2)]
+
+CREDENTIALS_LIST = [
+    Credentials.from_mapping(CREDENTIALS_DICT_1),
+    Credentials.from_mapping(CREDENTIALS_DICT_2),
+]
```

Comment:
Why create dictionaries and then use `from_mapping()`? Why not just create objects and use them?